### PR TITLE
Support recording conferences

### PR DESCRIPTION
**Pull request by n-miles**

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-4350.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/verbs/startRecording.md [hunk 1]
```diff
@@ -2,31 +2,33 @@
...
-event is sent to the URL once the recording is available for download, indicating the `mediaUrl` and if there was any issue processing the recording.
+or [Conference Recording Available](../callbacks/conferenceRecordingAvailable.md) event is sent to the URL once the
+recording is available for download, indicating the `mediaUrl` and if there was any issue processing the recording.
 BXML returned in response to this callback will be ignored.
 
-If the `transcriptionAvailableUrl` attribute is specified, then the [Transcription Available](../callbacks/transcriptionAvailable.md)
+If the `transcriptionAvailableUrl` attribute is specified for a call recording, then the [Transcription Available](../callbacks/transcriptionAvailable.md)
 event is sent to the URL once the transcription is available for download.
-BXML returned in response to this callback will be ignored.
+BXML returned in response to this callback will be ignored. Note that this attribute will be silently ignored if recording a conference.
```
Comment (marcelohossomi) [inline voice/bxml/verbs/startRecording.md:30]: Worth sticking to the same `<br><br>Note:` style you used in the other attributes?

File: voice/bxml/about.md [hunk 2]
```diff
@@ -70,7 +70,8 @@ BXML callbacks are HTTP messages that are sent to your application server to not
 | Callback                                                       | Description                                                                                                                      |
 |:---------------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------|
 | [Disconnect](callbacks/disconnect.md)                          | Bandwidth API sends this when a call ends.                                                                                       |
-| [Recording Available](callbacks/recordingAvailable.md)         | Bandwidth API sends this when a recording started by either a `<StartRecording>` or a `<Record>` verb is available for download. |
+| [ConferenceRecording Available](callbacks/conferenceRecordingAvailable.md) | Bandwidth API sends this when a recording started by a `<StartRecording>`in a conference is available for download.  |
```
Comment (marcelohossomi) [inline voice/bxml/about.md]:
```suggestion
| [Conference Recording Available](callbacks/conferenceRecordingAvailable.md) | Bandwidth API sends this when a recording started by a `<StartRecording>`in a conference is available for download.  |
```

**Comment by marcelohossomi** [comment]
You also have to include the new callback in voice/about.md and voice/bxml/callbacks/about.md

File: voice/bxml/verbs/startRecording.md (hunk 1 above)
Comment (marcelohossomi) [reply voice/bxml/verbs/startRecording.md:30]: Oh this is not in the table, nvm.

File: voice/bxml/verbs/conference.md [hunk 3]
```diff
@@ -23,7 +23,15 @@ A conference may last for at most 24 hours.
...
-Only `<PlayAudio>` and `<SpeakSentence>` are supported for conferences. BXML containing unsupported verbs will be rejected completely.
+Only the following verbs are valid for conferences:
+* [`PlayAudio`](playAudio.md)
+* [`SpeakSentence`](speakSentence.md)
+* [`StartRecording`](startRecording.md)
+* [`StopRecording`](stopRecording.md)
+* [`PauseRecording`](pauseRecording.md)
+* [`ResumeRecording`](resumeRecording.md)
+
+BXML containing unsupported verbs will be rejected completely.
```
Comment (marcelohossomi) [inline voice/bxml/verbs/conference.md:34]: There's a list of callbacks at the end of this file, might want to include the new callback there too.

Comment (n-miles) [reply voice/bxml/verbs/conference.md:34]: I reasoned against it on the grounds that it isn't this that produces it, but the `StartRecording` verb

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-tdp-4350.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/verbs/conference.md (hunk 3 above)
Comment (marcelohossomi) [reply voice/bxml/verbs/conference.md:34]: Oh, makes sense.

**Review by marcelohossomi — APPROVED** [review]

**Review by CatexB — APPROVED** [review]

**Pull request merged**